packages/menu/demo/~patterns/stories/NestedStory.tsx

```diff
@@ -0,0 +1,101 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useCallback, useRef, useState } from 'react';
+import { StoryFn } from '@storybook/react';
+import classNames from 'classnames';
+import { IUseMenuProps, IMenuItemBase, useMenu } from '@zendeskgarden/container-menu';
+import { BASE_ITEMS, NESTED_ITEMS } from './data';
+
+type UseMenuProps = IUseMenuProps<HTMLButtonElement, HTMLUListElement>;
+
+export const NestedStory: StoryFn<{ rtl: boolean }> = ({ rtl }) => {
+  const [args, setArgs] = useState<Pick<UseMenuProps, 'items' | 'focusedValue' | 'isExpanded'>>({
+    items: BASE_ITEMS,
+    focusedValue: undefined,
+    isExpanded: false
+  });
+  const triggerRef = useRef(null);
+  const menuRef = useRef(null);
+
+  const onChange = useCallback(({ type, isExpanded, focusedValue }) => {
+    const isNext = type.indexOf('next') > -1;
+    const isPrev = type.indexOf('previous') > -1;
+
+    if (isNext || isPrev) {
+      setArgs({
+        items: isNext ? NESTED_ITEMS : BASE_ITEMS,
+        focusedValue: isNext ? 'Strawberry' : 'Berry',
```

Comment:
Despite Garden's current [nested menu example](https://garden.zendesk.com/components/menu#nested) (which never received critical a11y scrutiny), I'm not feeling like this behavior maps to APG intent. Rather than `'Strawberry'` we should keep submenu focus on  `'Fruits'` – allowing the user to nav left/right back and forth without shifting focus to the item below `isPrevious`. Doing this sticks closer to https://www.w3.org/WAI/ARIA/apg/patterns/menubar/examples/menubar-navigation/#kbd_label